src/ddocs/ddocs.rst

```diff
@@ -116,17 +116,25 @@ single value - which could be an array or similar object.
 Built-in Reduce Functions
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Additionally, CouchDB has three built-in reduce functions. These are implemented
-in Erlang and run inside CouchDB, so they are much faster than the equivalent
-JavaScript functions: ``_sum``, ``_count`` and ``_stats``. Their equivalents in
-JavaScript:
+Additionally, CouchDB has a set of built-in reduce functions. These are
+implemented in Erlang and run inside CouchDB, so they are much faster than the
+equivalent JavaScript functions.
 
-.. code-block:: javascript
+.. data:: _approx_count_distinct
 
-    // could be replaced by _sum
-    function(keys, values) {
-        return sum(values);
-    }
+Since version `2.2.0`.
+
+Aproximates the number of distinct keys in a view index using a variant of the
+HyperLogLog algorithm. This algorithm enables an efficient, parallelizable
```

Comment:
Perhaps we can provide a link to an explanation of this algorithm for those (like me) who have never heard of it? Maybe just https://en.wikipedia.org/wiki/HyperLogLog ?

Reply:
Yes meant to do that thanks. Done.